packages/logger/src/Logger.ts

```diff
@@ -154,14 +153,6 @@ export class Logger {
      */
     private initStatus: boolean;
 
-    private static mProfileSchemas = new Map<string, IProfileSchema>();
-    public static get profileSchemas(): Map<string, IProfileSchema> {
-        return this.mProfileSchemas;
-    }
-    public static set profileSchemas(schemas: Map<string, IProfileSchema>) {
-        this.mProfileSchemas = schemas;
-    }
```

Comment:
Are these ok to remove without deprecating them first?

Reply:
Not sure why, the GitHub diff of new changes I hadn't seen before made it look like these methods were removed.